CI/build-mudlet-for-windows.sh

```
@@ -128,6 +130,93 @@ echo ""
+  BUILD_DIR="${GITHUB_WORKSPACE_UNIX_PATH}/build-${MSYSTEM}/${BUILD_CONFIG}"
+  echo "  Debug: Changing to build directory: ${BUILD_DIR}"
+  cd "${BUILD_DIR}" || exit 1
+  echo "  Debug: Successfully changed to: $(pwd)"
+else
+  echo "  Building without Sentry support"
+fi
+
+echo "Running qmake to make MAKEFILE ..."
+echo ""
+
+if [ "${MSYSTEM}" = "MINGW64" ]; then
```

Shellcheck (0.11.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)